src/main/java/io/appium/java_client/remote/MobileCapabilityType.java

```diff
@@ -134,4 +134,16 @@
      * or Safari's (on iOS) performance logging (default {@code false}).
      */
     String ENABLE_PERFORMANCE_LOGGING = "enablePerformanceLogging";
+
+
+    /**
+     * App or list of apps (as a JSON array) to install prior to running tests. Note that it will not work with
+     * automationName of Espresso and iOS real devices.
+     */
+    String OTHER_APPS = "otherApps";
```

Comment:
how about extracting this change into a separate PR?

Reply:
#1323